### ISSUE TITLE
Audit

### ISSUE THREAD

**Issue by andylolz** [pwyf/2018-technical-consultation#5]
## Description

The organisation’s annual audit of its activities is an official inspection of the accounts and activities of this organisation, typically by an independent body.

As previously, documents will be manually checked to verify that they contain the required information to score for the indicator.

## Proposed test

```python
`document-link/category[@code="B06"]` should be present
```

**Comment by abdulriza** [comment]
Audit document category is B06. Therefore, the test should read as `document-link/category[@code="B06"]`

**Comment by andylolz** [comment]
You’re quite right @abdulriza – thanks, and my apologies. This was a typo. I’ve now updated the original to reference `B06` instead of `B04`.

**Comment by AndieVaughn** [comment]
We recommend that organization file documents for CY and CY-1 are accepted for organizational file documents.  

There is a possibility that donors could score differently each year just based on the timing of the Index and when specific reports are published.  That penalty wouldn’t make sense. For example, if the cut-off for published data is December 31st and updated strategies come out in the spring, the reviewed strategy would be for the previous year and not the current year, even though another one would be in development and just short of publication.  Given fiscal years vary, a grace period of an additional 12 months makes sense when performing manual checks.

**Comment by andylolz** [comment]
Thanks for the comments here. This test will remain as proposed.

On the question of a grace period for organisation documents: Audits up to date with regular audit cycles are accepted, i.e. if the organisation publishes biennial audits, the most recent document within this time frame is accepted. Further details will be provided in the technical paper outlining the new methodology in its entirety.
